packages/amplify_auth_plugin_interface/lib/amplify_auth_plugin_interface.dart

```diff
@@ -80,7 +81,8 @@ abstract class AuthPluginInterface extends AmplifyPluginInterface {
     throw UnimplementedError('getCurrentUser() has not been implemented.');
   }
 
-  Future<List<AuthUserAttribute>> fetchUserAttributes({AuthUserAttributeRequest request}) {
+  Future<List<AuthUserAttribute>> fetchUserAttributes(
+      {AuthUserAttributeRequest request}) {
```

Comment:
Why are we changing just the formatting for this one method?  Maybe leave this for now, and fix all of them later if the formatting is wrong?

Reply:
I think it would be preferable to revert this, but if you feel strongly its ok.